docs/deploy-hyperlane.mdx

```diff
@@ -1,113 +1,334 @@
-5. [<b>Deploy a warp route</b>](#5-optional-deploy-a-warp-route) to send token value, not just messages, across chains
+To install the Hyperlane CLI npm package globally, run:
 
-## Getting Started
+```bash
+npm i -g @hyperlane-xyz/cli
+```
 
-## 1. Set up keys
+Verify the expected version is installed via:
 
-<SetupKeysPartial />
+```bash
+hyperlane --version
+```
 
-## 2. Deploy contracts
+To view a list of all available command, run:
 
-<DeployContractsPartial />
+```bash
+hyperlane --help
+```
 
-## 3. Run validators and relayer
+## 2) Registry: add your custom chain
 
-:::note
+Let’s create a custom chain config, run:
 
-This section covers how to 1-click deploy agents using a third party. To run the Hyperlane agent binaries yourself, follow the [Local Agents guide](guides/deploy-hyperlane-local-agents.mdx).
+```bash
+hyperlane registry init
+```
 
-:::
+Follow the prompts to set up your RPC URL, chain name, and chain ID. Setting up block or gas properties is optional. Once successful, you should see this output:
 
-We've partnered with [Kurtosis](https://www.kurtosis.com/) to provide a single-click validator and relayer deployment in the cloud. You'll receive a one-month trial during which your validator and relayer run in Kurtosis Cloud for free. After your trial ends, you'll need to either run your validator and relayer yourself or purchase a Kurtosis Cloud subscription.
+```tsx
+Skipping updating chain yourchain at github registry
+Now updating chain yourchain at filesystem registry at $HOME/.hyperlane
+Done updating chain yourchain at filesystem registry
+```
 
-To get started, use the `deploy kurtosis-agents` command to generate a link for configuring your Kurtosis deployment.
+Now, under `$HOME/.hyperlane/chains` you will find a new folder named with your custom chain’s name, and a file named `metadata.yaml` within that folder.
+
+<details> 
+  <summary>
+    Example content of this config at `$HOME/.hyperlane/chains/yourchain/metadata.yaml`
+  </summary>
+
+   ```yaml
+    # yaml-language-server: $schema=../schema.json
+    name: yourchain
+    chainId: 171717
+    domainId: 171717
+    protocol: ethereum
+    rpcUrls:
+      - http: https://hyper-lane-docs.rpc.caldera.xyz/http
+    nativeToken:
+      name: Ether
+      symbol: ETH
+      decimals: 18
+    ```
+</details>
+
+## 3) Core: configure & deploy your custom chain’s core contracts
+
+Next, let’s configure, deploy and test your custom chain’s core contracts.
+
+### Initialize configuration
+
+1. From your local environment, source the private key or seed phrase of your funded deployer address via `HYP_KEY`. For example: `export HYP_KEY='<YOUR_PRIVATE_KEY>'`
+2. Verify your `HYP_KEY` is sourced correctly via `echo $HYP_KEY`
+3. From the same instance, run:
+    
+  ```bash
+  hyperlane core init
+  ```
+<details>
+  <summary>
+    After successfully creating a new core deployment config, you can find it in a file named `core-config.yaml` within  `./configs`
```

Comment:
Sorry if it feels like I'm nitpicking here but I'm finding this updated deploy guide more verbose than it needs to be in a lot of places. Brevity is good! Like I'd expect this to say just: `The deployment config will be written to './configs/core-config.yaml'`

Reply:
fixed.